src/plugins/legacy.js

```diff
@@ -38,7 +38,7 @@ export type LegacyCommand = {
 }
 
 export function convertFromV5 (c: LegacyCommand) {
-  if (!c.topic) throw new Error('command has no topic')
+  if (c.topic == null) throw new Error('command has no topic')
```

Comment:
ah I see, can you change it to `if (cmd !== '')` then? I think that's more clear

```
node -e "let types = [undefined, null, '', 'string']; types.forEach(t => console.dir({[t]: t !== '' ? true : false}))"
{ undefined: true }
{ null: true }
{ '': false }
{ string: true }

$ node -e "let types = [undefined, null, '', 'string']; types.forEach(t => console.dir({[t]: t == null ? true : false}))"
{ undefined: true }
{ null: true }
{ '': false }
{ string: false }
```